js/controllers/aboutController.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+(function(module){
+  var aboutController = {};
+
+  aboutController.index = function(){
+    $('.tab').removeClass('active');
+    $('li[data-content="about-me"]').addClass('active');
```

Comment:
I am not sure that I want to do chaining here, I want to clear all .tab elements that may have the active class to make sure nothing has active before I assign it.